### PR TITLE
update resource editor

### PR DESCRIPTION
**Pull request by jacobvenable**
This PR does not close an issue.

## What does this PR do?

- updates the layout of the schedule in the resource editor
- updates use of "Resource" to "Provider" in various parts of the application
- moves the "Trash Resource" option to the bottom of the editor page

## Screenshots

scheduler

## How does this PR make you feel? [:link:](http://giphy.com/categories/emotions/)

that feeling you get when you make planning easier

### CODE REVIEW

File: packages/upswyng-server/src/components/ResourceEditor.svelte [hunk 1]
```diff
@@ -80,7 +80,7 @@
     {/if}
     {#if resource.resourceId}
       <p>
-        <span class="label">Resource ID</span>
+        <span class="label">Provider ID</span>
```
Comment (rhinodavid) [inline packages/upswyng-server/src/components/ResourceEditor.svelte:83]: I think we need to make sure all occurrences say "Service Provider"

File: packages/upswyng-server/src/components/ScheduleSelector.svelte [hunk 2]
```diff
@@ -259,132 +258,122 @@
...
-        <div class="content">
-          <div class="field">
-            <label class="label">
-              Certain occurances of the day in a month
-            </label>
-          </div>
-          <div class="field has-addons">
+      {#if !certainWeekRepeatState.everyWeek}
+        <div class="field">
+          <label class="label">How often do these days repeat?</label>
```
Comment (rhinodavid) [inline packages/upswyng-server/src/components/ScheduleSelector.svelte:301]: This label doesn't make sense.

Q: "How often do these days repeat?"
A: "Third"

🤯

Comment (jacobvenable) [reply packages/upswyng-server/src/components/ScheduleSelector.svelte]: Ya that's fair. What about this [update](https://github.com/CodeForBoulder/upswyng/pull/333/commits/d3ec52e76cec9d76606149c167a7f3e16e09bac9)?
Screen Shot 2020-05-06 at 11 26 12 AM

File: packages/upswyng-server/src/components/ResourceEditor.svelte (hunk 1 above)
Comment (jacobvenable) [reply packages/upswyng-server/src/components/ResourceEditor.svelte]: 👍 [updated](https://github.com/CodeForBoulder/upswyng/pull/333/commits/8fac2ab788a6f081cde6ab8535f499a42038d3ae)

File: packages/upswyng-server/src/components/ScheduleSelector.svelte (hunk 2 above)
Comment (rhinodavid) [reply packages/upswyng-server/src/components/ScheduleSelector.svelte]: That looks pretty good. I just remembered I pulled some screenshots from google calendar I'll post them up. This seems better though

**Pull request merged**